packages/components/package.json

```diff
@@ -52,8 +52,11 @@
 		"@wp-g2/utils": "^0.0.160",
 		"classnames": "^2.2.5",
 		"create-emotion": "^10.0.27",
+		"deepmerge": "^4.2.2",
 		"dom-scroll-into-view": "^1.2.1",
 		"downshift": "^6.0.15",
+		"emotion": "^11.0.0",
```

Comment:
Is this one different from `@emotion/core` should we align?

Reply:
I think I have the version number wrong here, but in Emotion 10 `cx` comes from `emotion` instead of `@emotion/core` 🙁